public/js/script.js

```diff
@@ -15,34 +17,43 @@ $(document).ready(function() {
     }
 
     $('#question').html(data.question.question);
+    if (data.question.id !== defaultQuestionId) {
+      $("#answer-input").prop('disabled', false);
+      $('#answer-input').attr('placeholder', 'post an answer');
```

Comment:
The answer field is now disabled by default, to prevent users from attempting to post answers if there is no question. This code will re-enable it on load, if it doesn't load the default question.